pkg/resources/view_grant.go

```diff
@@ -111,37 +108,34 @@ func CreateViewGrant(data *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
-	// ID format is <db_name>|<schema_name>|<view_name>|<privilege>
-	dataIdentifiers := make([][]string, 1)
-	dataIdentifiers[0] = make([]string, 4)
-	dataIdentifiers[0][0] = dbName
-	dataIdentifiers[0][1] = schemaName
-	dataIdentifiers[0][2] = viewName
-	dataIdentifiers[0][3] = priv
+	// ID format is <db_name>|<schema_name>|<view_name>|<privilege> or <db_name>|||<privilege>
```

Comment:
hmm, why did you add the `or <db_name>|||<privilege>` part to this comment? I might be missing something